src/pages/ReviewPage.tsx

```diff
@@ -344,8 +346,8 @@ class ReviewPage extends React.Component<RouteComponentProps, State> {
                         />
                       )}
                     </ContestProse>
-                    <ContestActions>
-                      <DecoyButton>Change</DecoyButton>
+                    <ContestActions aria-label="press enter to change your answer for this contest.">
+                      <DecoyButton aria-hidden="true">Change</DecoyButton>
```

Comment:
Why not place the label on the DecoyButton? Seems like that's where the "action" is? 😃

Reply:
Because if I do that, it says "button" and I can't stop it from saying that :)

Reply:
Because if I do that, it says "button" and I can't stop it from saying that :)